### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by dOrgJelli** [dOrgTech/arc.react#15]
For each component, have auto-generated docs (description, props, data, code, contextual info) alongside examples and more detailed information (if needed). Ideally this could all be auto-generated so nothing too crazy.

**Comment by dOrgJelli** [comment]
Hand authoring this may be the easiest way, would only take a few hours I think :P

**Comment by dOrgJelli** [comment]
Documentation in the DAOcomponents README (or whatever the most accessible documentation) should have:
* DAOcomponents definition – what it is and what you can do with it  
* Local environment setup – working instructions for getting a local instance of a sample DAOcomponents DAO UI up and running  
* Architecture – the documentation should include at least a basic explanation of how it works beyond the sample app, for anyone who wants to go deeper than the example  
* Sales pitch – blurb explaining our custom DAO UI service and contact information for DAOstack or dOrg (we can provide this copy)

**Comment by dOrgJelli** [comment]
Developer documentation should outline how to add new "Components" and "ComponentLists" to the project.
